code/game/machinery/cloning.dm

```diff
@@ -17,6 +17,7 @@
 	req_access = list(ACCESS_CLONING) //FOR PREMATURE UNLOCKING.
 	verb_say = "states"
 	circuit = /obj/item/circuitboard/machine/clonepod
+	var/shorted = FALSE //Cut wires

```

Comment:
It would probably be easier to check `wire.is_cut` rather than having a second variable that is kept up to date